server/api.js

```diff
@@ -1,9 +1,19 @@
 import { Router } from "express";
+import db from "./db";
 
 const router = new Router();
 
-router.get("/", (_, res) => {
+router.get("/", (req, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
+router.get("/clients", (req, res) => {
+	db.query("SELECT * from clients;")
```

Comment:
```suggestion
	db.query("SELECT * from clients")
```
Semi-colons aren't needed in the queries here, so you might find that various software teams prefer not to include them